hasher-matcher-actioner/hmalib/lambdas/api/middleware.py

```diff
@@ -154,3 +154,17 @@ def wrapper(*args, **kwargs):
             return json.dumps(body.to_json())
 
         return wrapper
+
+
+class SubApp(bottle.Bottle):
+    def __init__(self):
```

Comment:
Typically, when extending 3rd party classes, use __init__(self, *args, **kwargs) and pass them to the super().__init__ call.

[Bottle](https://github.com/bottlepy/bottle/blob/master/bottle.py#L632) seems to support some kwargs.